src/App.js

```diff
@@ -1,23 +1,56 @@
-import logo from './logo.svg';
-import './App.css';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { Route, Routes, useLocation } from 'react-router-dom';
+import Login from './Components/Login/Login';
+import CarDetails from './Components/Car/CarDetails';
+import SideBar from './Components/Layout/SideBar';
+import AddForm from './Components/Car/CarForm';
+import ReservationForm from './Components/Reservation/ReservationForm';
+import Car from './Pages/Cars';
+import Thankyou from './Pages/Thankyou';
+import Sorry from './Pages/Sorry';
+import Reservations from './Pages/Reservations';
+import DeleteCar from './Pages/DeleteCar';
+import { fetchCars } from './Redux/Car/carSlice';
 
 function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 5): 

![image](https://github.com/batoolfatima2135/Driveasy/assets/57421823/f26acc42-e211-49e7-9983-59fded46425e)

That way, your code will meet the latest standards. 💪